Dockerfile.UT

```diff
@@ -0,0 +1,14 @@
+FROM phusion/baseimage:0.11 as builder
+LABEL maintainer="team@cere.network"
+LABEL description="This is the build stage to create the binary."
+ARG PROFILE=release
+WORKDIR /cerenetwork
+COPY . /cerenetwork
+
+RUN apt-get update && \
+	apt-get upgrade -y && \
```

Comment:
Align spaces. As an example take a look to [this](https://github.com/Cerebellum-Network/pos-network-node/blob/dev-cere/Dockerfile#L8).